### DISCUSSION TITLE
ランキングに直結しない、気軽に押せる「いいね」ボタンがあった方がいいのでは

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#126] [要望・アイデア・不具合]
現在、QiitaではLGTMとストックの機能があります。僕の中では使い分けはこんなイメージです。

- LGTM = 役に立った、内容が正しい、と思ったときにクリック。獲得数が多いとトレンドに上がったり、ランキングメールに載ったりする。投稿者のContribution数にも反映される
- ストック = あとで何度も読み返したいと思った記事でクリック。ランキングメールには影響しないが、投稿者のContribution数には反映される。

参考 

- [Contribution計算に含める項目を追加します - Qiita Blog](https://blog.qiita.com/add-items-for-calculating-contribution/)
- [Qiitaの「いいね」が「LGTM」に変わります - Qiita Blog](https://blog.qiita.com/like-to-lgtm/)

一方で、僕はこんなユースケースもよくあるんじゃないかなーと思っています。

- 会社で新人のAさんが記事を書いたので、「がんばったね！」「いいね！」「グッジョブ！」の意味で、同僚がLGTMをクリックする

が、たぶん、これは本来のLGTMの使い方ではないはずです。さらに、その会社の規模が大きいと、多くの社員がLGTMをクリックするので、自ずとトレンドに載ったり、投稿者やその会社のContribution数も上がったりしてしまいます。

結局、今のQiitaには「がんばったね！」「いいね！」「グッジョブ！」を投稿者にフィードバックする手段がないので、手っ取り早くクリックできるLGTMがクリックされるのではないかと思います。もともと「LGTM=いいね」だったわけですし。
ですが、そんな「がんばったね！」「いいね！」「グッジョブ！」の意味のLGTMがランキングやContribution数（＝記事や投稿者、所属する会社の評価）に直結してしまうのはどこかしっくりこない、不公平感のようなものを感じます。

なので、ランキングに直結しない、気軽に押せる「いいね」ボタンがあった方がいいのでは、と思ったのですが、いかがでしょうか？

もちろん、LGTM、ストック、いいねと3つのボタンが出てくると、使い分けが難しくなり、使い分けを理解できなくて結局いいねの意味でLGTMをクリックしてしまう、というケースも出てくると思いますが・・・。

とはいえ、評価に直結しない「がんばったね！」「いいね！」「グッジョブ！」のフィードバックを残せる仕組みは何かあったほうが便利なんじゃないかな〜と思います。

**Comment by JunichiIto** [comment]
追伸、

イメージ的には、はてなスターが近いかもしれません。
いい！と思ったら連打できる、でもいくら押されても評価には直結しない、みたいな感じです。

<img width="677" alt="Screen Shot 2022-02-01 at 9 21 50" src="https://user-images.githubusercontent.com/1148320/151894599-b4f71b55-ee86-48f5-8411-94e359200f64.png">

**Comment by JunichiIto** [comment]
はてなスターでなくても、GitHubのフィードバックボタンのようなものでもいいですね。（ネガティブ系のボタンはなくすとして）

<img width="383" alt="Screen Shot 2022-02-01 at 9 32 52" src="https://user-images.githubusercontent.com/1148320/151895441-a9304444-7305-45a6-89f7-bdcda0b71013.png">

<img width="227" alt="Screen Shot 2022-02-01 at 9 34 57" src="https://user-images.githubusercontent.com/1148320/151895601-ed18c256-aecc-4a40-a8ee-3811e7fd3874.png">

**Comment by kskwtnk** [comment]
@JunichiIto 

ご意見ありがとうございます！

> 会社で新人のAさんが記事を書いたので、「がんばったね！」「いいね！」「グッジョブ！」の意味で、同僚がLGTMをクリックする

現行のLGTMの意味づけからは少しずれているものの、こういった押され方は多いと思われますし、感情としても非常に理解できます
また、先月末のオンラインミートアップでも @getty104 から少し共有させていただきましたが、我々としても「気軽な賞賛」があった方が良いと考えています

というのもあり、気軽に「いいね」の気持ちを示せるインターフェースは検討しています！
ただ、LGTMへ変更してから2年程度しか経っておらず、混乱が起きないようにコミュニケーションする必要はありそうで、若干慎重な進め方になるのではないか……と思っています 🙏

**Reply by robozushi10** [reply]
(古い体質の会社ということもあって) 職場で Qiita を見るときは、Qiita にログインできません.
そのために、非常に参考になった記事に対して、御礼の意味での LGTM を押すことができません.

ログインしなくても押せるような、代わりのボタンが欲しいです.
また、検索のためにも、ボタンの押下数によるソート機能は欲しいです.

なお、押下数によるランキングは要らないです.

**Reply by munierujp** [reply]
FYI: noteでは、ログインしていなくても「スキ」が押せますね。

cf. [noteにログインしていなくてもスキができるようになりました｜note公式｜note](https://note.com/info/n/nb4bcf9eb079e)

**Reply by kskwtnk** [reply]
@robozushi10 @munierujp 
ありがとうございます！

こちらのDiscussionの件もあり

- ログインしていなくても押せる
- トレンドやランキングに関係ない仕組みとして何回でも押せる

など、検討中です！

**Comment by JunichiIto** [comment]
> というのもあり、気軽に「いいね」の気持ちを示せるインターフェースは検討しています！
ただ、LGTMへ変更してから2年程度しか経っておらず、混乱が起きないようにコミュニケーションする必要はありそうで、若干慎重な進め方になるのではないか……と思っています 🙏

了解です！ご検討いただけるとありがたいです。

ちなみに最初の投稿で「どこかしっくりこない、不公平感のようなものを感じます」と書きましたが、これは他の人に対する感情だけでなく、自分がQiitaを使うときでもつい、「いや、ここは本当はLGTMじゃないんだよな〜」という後ろめたい気持ちを持ちながら「いいね！」の意味のLGTMをクリックすることがある、ということを追記しておきます😅

**Reply by tanaka-asami415** [reply]
@JunichiIto
「LGTM」を「いいね」に戻しましたので、ご報告させていただきます。
https://github.com/increments/qiita-discussions/discussions/283

戻すにあたり、ランキングに直結しないことなども検討したのですが、
一旦シンプルに「LGTM」を「いいね」に戻すことにいたしました。
今後は、がんばったも含めて「いいね」を使ってもらえたらいいなと考えております🙏

また、これで完璧！というわけではなく、戻したことによる変化なども見て
ログインしていなくても押せる方法や、contributionと直結しない方法など、
引き続きより良い方法を模索していきたいと思っています💪
